Update contributing.en.md with adding emphasis to Hugo version to make it look visible

naeemakram opened the pull request:
Mention Node.js and postcss-cli requirement
Added a line of text mentioning the Node.js library and postcss-cli which are also needed to build and run the site locally.

**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to review and merge it quickly**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, and help reviewers by making them as simple and short as possible.


<!--- Provide a general summary of your changes in the Title above -->

### Description
<!--- Describe your changes in detail -->

### Motivation and Context
<!--- Why is this change required? What problem does it solve? -->

### Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [x] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [ ] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.
<!--- Provide a general summary of your changes in the Title above -->

naeemakram committed:
Update contributing.en.md

Mention Node.js and postcss-cli requirement
Added a line of text mentioning the Node.js library and postcss-cli which are also needed to build and run the site locally.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/SeleniumHQ/seleniumhq.github.io?pullRequest=1150) <br/>All committers have signed the CLA.

diemol commented:
If you are using the extended version of Hugo, this is not needed.

If we were to add this, it should be in the shape of the specific command in step 3.

naeemakram commented:
I am a new contributor, it took me a while to figure these things out. I think if such small details are mentioned somewhere, onboarding new contributors will be easier.
What are the alternatives of using Hugo extended version? 
Shall I change the verbiage and mention "when you're using Hugo"?

diemol commented:
> I am a new contributor, it took me a while to figure these things out. I think if such small details are mentioned somewhere, onboarding new contributors will be easier. What are the alternatives of using Hugo extended version? Shall I change the verbiage and mention "when you're using Hugo"?

I thought it was clear when we mention it in this link https://www.selenium.dev/documentation/about/contributing/#dependencies-hugo, during step 1. Do you think it should be shown elsewhere?

naeemakram commented:
When I was reading this page I didn't find a mention of Node.js anywhere. 
I also missed the word "extended" and overall it took me an hour or so to generate live preview of the site.
I suggest the following
1. Make it clear in what scenarios Node.js is needed
2.  highlight the word "extended" in the sentence below. It can be made bold black or bold red. 
3. 
You will need the “extended” Sass/SCSS version of the Hugo binary to work on this site.

diemol commented:
> When I was reading this page I didn't find a mention of Node.js anywhere. I also missed the word "extended" and overall it took me an hour or so to generate live preview of the site. I suggest the following
> 
>     1. Make it clear in what scenarios Node.js is needed
> 
>     2. highlight the word "extended" in the sentence below. It can be made bold black or bold red.
> 
> 
> You will need the “extended” Sass/SCSS version of the Hugo binary to work on this site.

Did you need Node to run the site? The idea of using the extended version of Hugo is to avoid having Node as a requirement. 

Would you like to change this PR to follow your suggestion?

naeemakram commented:
I will make any changes needed, please let me know. This is my first PR, any guidance will help. 

About Node, I will remove Node from my PC and try again. Because I found mention of Node on some threads while trying to resolve the problem.
I will confirm the Node requirement when I get back home

naeemakram committed:
Update contributing.en.md

Added emphasis around the words extended SASS/SCSS
Use upper-case in word SASS instead of lower-case

naeemakram commented:
Node.JS is not needed; I've removed the line which I added.
I have added an emphasis around the words "extended" SASS/SCSS and capitalized the word SASS which was previously in lower case.

diemol commented:
Can you please fix the conflicts so we can get this merged?

naeemakram commented:
> **jovial-austin-42fe02 ** Completed in 1m — No redirect rules processed
> [Details](/SeleniumHQ/seleniumhq.github.io/pull/1150/checks?check_run_id=8099404518)
> [ <img alt="@netlify" width="20" height="20" src="https://avatars.githubusercontent.com/in/13473?s=40&amp;v=4"> ](/apps/netlify)
> ** Mixed content - jovial-austin-42fe02 ** Successful in 1m — No mixed content detected
> [Details](/SeleniumHQ/seleniumhq.github.io/pull/1150/checks?check_run_id=8099404619)
> [ <img alt="" width="20" height="20" src="https://avatars.githubusercontent.com/oa/160438?s=40&amp;u=51ee531f5a74a9839fb9c7c43832494cedf1aae5&amp;v=4"> ](https://cla-assistant.io)
> ** license/cla ** — Contributor License Agreement is signed.

Sure, I will resolve the conflicts later today. I am sorry I am at my job, away from my personal machine where I've setup this repo.

naeemakram commented:
I've resolved the conflict, I updated the Hugo version number to the latest one in my text.

naeemakram committed:
Update contributing.en.md

Updated required Hugo version number to 0.101.0 to resolve merge conflict

naeemakram committed:
Update contributing.en.md

Changed line breaks in source file.

naeemakram commented:
I've made changes to my commit and pushed again. Please note that I don't have the rights to resolve the conflicts on GitHub.

titusfortner commented:
The recommended version of Hugo was updated last week, so this PR still has a conflict. Please rebase with latest version and update.

naeemakram commented:
I think I should redo the change on a fresh branch instead of letting this one hang. It is a tiny change after-all.
